api/nodes/views.py

```diff
@@ -659,9 +659,9 @@ class NodeDraftRegistrationDetail(JSONAPIBaseView, generics.RetrieveUpdateDestro
     Use DraftRegistrationDetail endpoint instead.
     """
     permission_classes = (
+        DraftRegistrationPermission,
```

Comment:
Given everything else I've said, I *think* the Detail views can go untouched for now.

Reply:
Do we still need to update the permission of the detail view?